### PR TITLE
LPS-140269 Add conditions when the objectDefinitonId 1 and 2 are equals

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-140269

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:relevant

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java [hunk 1]
```diff
@@ -133,6 +134,50 @@ public void testAddObjectRelationship() throws Exception {
 		Assert.assertFalse(_hasTable(objectRelationship.getDBTableName()));
 	}
 
+	@Test
+	public void testAddObjectRelationshipMappingTableValues() throws Exception {
+		String name = StringUtil.randomId();
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:139]: The variable name is only used 1 time. Could you put the code directly where it is going to be used, please?

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java [hunk 2]
```diff
@@ -133,6 +134,50 @@ public void testAddObjectRelationship() throws Exception {
...
+			objectRelationship.getUserId(),
+			objectRelationship.getObjectRelationshipId(),
+			_objectDefinition1.getPrimaryKey(),
+			_objectDefinition1.getPrimaryKey(), null);
+
+		List<ObjectRelationship> objectRelationshipList =
+			_objectRelationshipLocalService.getObjectRelationships(
+				_objectDefinition1.getObjectDefinitionId(), 0, 10);
+
+		System.out.println(objectRelationshipList);
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:160]: Could you remove this line of code, please?

**Review by marcelabc — CHANGES_REQUESTED** [review]

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:forward

**Comment by marcelabc** [comment]
ci:stop

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java [hunk 3]
```diff
@@ -133,6 +134,47 @@ public void testAddObjectRelationship() throws Exception {
...
+			objectRelationship.getUserId(),
+			objectRelationship.getObjectRelationshipId(),
+			_objectDefinition1.getPrimaryKey(),
+			_objectDefinition1.getPrimaryKey(), null);
+
+		List<ObjectRelationship> objectRelationshipList =
+			_objectRelationshipLocalService.getObjectRelationships(
+				_objectDefinition1.getObjectDefinitionId(), 0, 10);
+
+		for (ObjectRelationship relationship : objectRelationshipList) {
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:159]: Could you rename this var to "objectRelationship", please? 
Sorry for the delay in reviewing this part! :/

**Review by marcelabc — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java [hunk 4]
```diff
@@ -133,6 +134,47 @@ public void testAddObjectRelationship() throws Exception {
...
+				StringUtil.randomId(),
+				ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
+
+		_objectRelationshipLocalService.addObjectRelationshipMappingTableValues(
+			objectRelationship.getUserId(),
+			objectRelationship.getObjectRelationshipId(),
+			_objectDefinition1.getPrimaryKey(),
+			_objectDefinition1.getPrimaryKey(), null);
+
+		List<ObjectRelationship> objectRelationshipList =
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java:155]: Could you rename this var to "objectRelationships", please?
Sorry for the delay in reviewing this part! :/

**Comment by marcelabc** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:forward

**Comment by danvb112** [comment]
ci:forward

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**